### DISCUSSION TITLE
How to cite SmartNoise in academic paper?

### DISCUSSION THREAD

**Discussion by shlomihod** [opendp/opendp#640] [SmartNoise] [answered]
Should I refer to the GitHub repo?

**Comment by Shoeboxam** [comment, marked as answer]
Are you citing SmartNoise or OpenDP? 

* Smartnoise: cc @joshua-oss 
* OpenDP: We have a citation.cff, so there is a cite button in the right panel of https://github.com/opendp/opendp!
